### ISSUE TITLE
Chinese path in JavaAgent have some encoding problem

### ISSUE THREAD

**Issue by Nsiso** [eclipse/openj9#6722]
Hello
I'm using openj9 to launch the minecraft (funny...)
when I'm using the JavaAgent to create a jvm, the problem appear:
the command line is like this:
javaw.exe -javaagent :"E:\MC\^桌面\华拥\华拥客户端\xxxxxxxx\xxxxxx.jar"=xxxxxxxx ...(and many other args)
you will see the path is including chinese characters
After the launch, the jvm will return the error message:
![TIM图片20190811012159](https://user-images.githubusercontent.com/27906817/62824937-85946080-bbd6-11e9-901e-c35100fb6359.png)
I noticed that the chinese characters all turn to a mess...
I guess it might be some encoding problem
Oracle java is ok, Only openj9.

Kind regards,
Siso

**Comment by Nsiso** [comment]
Error message I didn't intercept complete, In front of the error is:
Error opening zip file or JAR manifest missing : [the messy path]

**Comment by JasonFengJ9** [comment]
could you provide a simplified command line showing that Oracle Java works but OpenJ9 doesn't?

I think there was a known issue about Windows wide character encoding support in Java launcher which is common to both JVM though.

**Comment by Nsiso** [comment]
just simple javaagent with Chinese characters path.
For example, create a folder named "测试文件夹", just make sure Chinese characters included, and put a jar in it (or just change the jar into Chinese name like "测试.jar"?)
and load the jar in javaagent
and you will see the error

I make sure that just Openj9 have this problem, no oracle.
we test it with friends and it is true.

**Comment by JasonFengJ9** [comment]
Okay, will give a try then.

**Comment by JasonFengJ9** [comment]
**Update:**

Built a simple java agent jar and a hello world application (attached), it works with ascii path/file names. `java -showversion -javaagent:c:\ive\jars\javaagent\agent.jar -jar c:\ive\jars\javaagent\HelloWorld.jar` has following output:
```
Hello from agent premain!
java version "1.8.0_51"
Java(TM) SE Runtime Environment (build 1.8.0_51-b16)
Java HotSpot(TM) 64-Bit Server VM (build 25.51-b03, mixed mode)

Hello world!
```

However HotSpot JVM doesn't work with Chinese character in the path such as 
```
java -showversion -javaagent:C:\ive\jars\1测试\目录\agent.jar -jar c:\ive\jars\javaagent\HelloWorld.jar
Error opening zip file or JAR manifest missing : C:\ive\jars\1??\??\\agent.jar
Error occurred during initialization of VM
agent library failed to init: instrument
```
In addition, renaming `agent.jar` to `代理.jar` doesn't work with an ascii name path either.
```
java -showversion -javaagent:c:\ive\jars\javaagent\代理.jar -jar c:\ive\jars\javaagent\HelloWorld.jar
Error opening zip file or JAR manifest missing : c:\ive\jars\javaagent\??.jar
Error occurred during initialization of VM
agent library failed to init: instrument
```
HotSpot and OpenJ9 have same behaviours. The environment I tested was HotSpot JVM Java 8 at Windows 10 enterprise with language set to Chinese.

[HelloWorld.zip](https://github.com/eclipse/openj9/files/3538741/HelloWorld.zip)
[agent.zip](https://github.com/eclipse/openj9/files/3538742/agent.zip)

@Nsiso what's your environment? OS, Java, etc., specifically what's the output of `chcp` at the windows console?

**Comment by Nsiso** [comment]
OS is using windows 10 pro. with Chinese set and same java8.
chcp output is 936 (GBK)

**Comment by Nsiso** [comment]
Yeah, the test is passed and running smoothly.
![TIM截图20190827123033](https://user-images.githubusercontent.com/27906817/63741385-76e0c580-c8c7-11e9-9826-061c55b3e026.png)
using jre1.8.0_171.
I am wondering is that the default encoding makes the error?
Win10 in china is using GBK default encoding.

**Comment by JasonFengJ9** [comment]
A quick update:

Not able to make `RI` work even when the encoding is set to `GBK` just like `OpenJ9` so can't reproduce the behaviour difference reported. There are some environment difference between mine and the originator's which need to be identified.

**Comment by pshipton** [comment]
Seems this is stalled and won't be fixed for 0.17 so I'll move it to the next release.
